From 7df55bcbf4f7534be9913e53513bf0b05eaba10f Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 13 Apr 2021 19:05:44 +0300
Subject: [PATCH] [SW-0] gaudi: rma_info: improve spi_copy() message

Current code prints 2 times the same message which does not provide
usable enough information.
Make the message more informative.

Change-Id: I917733927502a8341bb554abfa110c621a61e92b
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/gaudi/rma_info.c b/gaudi/rma_info.c
index 629db1f..c6ccf15 100644
--- a/gaudi/rma_info.c
+++ b/gaudi/rma_info.c
@@ -73,10 +73,10 @@
 
 	/* on every read we first choose a valid sector to read from */
 	spi_copy(&header0, flash_mem_context.base[0],
-				sizeof(struct flash_header), "Flash sector validity");
+				sizeof(struct flash_header), "RMA header0");
 
 	spi_copy(&header1, flash_mem_context.base[1],
-				sizeof(struct flash_header), "Flash sector validity");
+				sizeof(struct flash_header), "RMA header1");
 
 	if (header0.magic == FLASH_SECTOR_MAGIC &&
 	    header1.magic == FLASH_SECTOR_MAGIC) {
